src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseFVM.cpp

```diff
@@ -444,6 +444,14 @@ void SinglePhaseFVM< BASE >::applyFaceDirichletBC( real64 const time_n,
                          string const & )
   {
     BoundaryStencil const & stencil = fluxApprox.getStencil< BoundaryStencil >( mesh, setName );
+    if( fs.getLogLevel() >= 2 && m_nonlinearSolverParameters.m_numNewtonIterations == 0 )
+    {
+      localIndex const numTargetFaces = MpiWrapper::sum( stencil.size() );
+      GEOSX_LOG_RANK_0( GEOSX_FMT(
+                          "SinglePhaseFVM {}: at time {}s, the <{}> boundary condition named \"{}\" is applied to the face set named \"{}\" in \"{}\". \nThe total number of target faces (including ghost faces) is {}. \nNote that if this number is equal to zero, the boundary condition will not be applied on this face set.",
+                          this->getName(), time_n+dt, AquiferBoundaryCondition::catalogName(), fs.getName(), setName, targetGroup.getName(), numTargetFaces ) );
```

Comment:
Same comment here about outputting the value of the BC.

Reply:
In single-phase flow as well, I output the scale of the boundary condition for SourceFlux and Dirichlet BCs. 

For Aquifer, the flux computation is quite convoluted, but yes it would definitely be useful to output the cumulative aquifer flux from [here](https://github.com/GEOSX/GEOSX/blob/bb63df344b130034593a4163a58be6a55184a955/src/coreComponents/physicsSolvers/fluidFlow/FlowSolverBase.cpp#L488) for debugging purposes. I will add that.